src/App.tsx

```diff
@@ -115,7 +114,7 @@ function app(): JSX.Element {
         <Route exact path="/forgotPassword" component={ForgotPassword} />
 
         {/* User Portal Routes */}
-        <Route exact path="/user" component={UserLoginPage} />
+        <Route exact path="/user" component={LoginPage} />
```

Comment:
@akhilender-bongirwar you can remove this path, one single path for the Login Page is sufficient.

Reply:
@rishav-jha-mech, in the `LoginPortalToggle` component, we are switching to the `/user` route. I have utilized it to set the role as the default user when the route is `/user`. Currently, it is necessary for the functionality.

Reply:
However, do you want me to remove the `LoginPortalToggle` component and instead add it to the Login Page? This way, we can switch roles within a single path for the Login Page.

Reply:
@akhilender-bongirwar a new endpoint is not required for that, you can remove it, by default let admin be selected for now

Reply:
@rishav-jha-mech  I apologize for the delayed reply, as I had mid-semester exams. I have made the necessary changes to remove the `/user` endpoint and set admin as the default selection. Please review : )